src/base/entriesScreen.js

```diff
@@ -0,0 +1,70 @@
+const match = require('../match');
+const { Screen } = require('./screen');
+
+
+class EntriesScreen extends Screen {
+    /**
+    * @param {object} parentScreen
+    * @param {string} parentScreen.openLabel
+    * @param {string} parentScreen.backLabel
+    * @param {object} parentScreen.screenTxt
+    * @param {object} parentScreen.generalTxt
+    * @param {object} config
+    */
+    constructor(parentScreen, config) {
+        super(parentScreen);
+        this.config = config;
+    }
+    InfoLabel() {
+        return match.accessible.Label(this.screenTxt.Info);
```

Comment:
[nit] `Txt` -> `Text` (not sure if saving one character is worth it)

Reply:
lol - yeah